blockly-core/core/ui/block_svg/block_svg_functional.js

```diff
@@ -51,7 +51,7 @@ Blockly.BlockSvgFunctional.prototype.renderDraw_ = function(iconWidth, inputRows
 
   this.blockClipRect_.setAttribute('d', this.svgPath_.getAttribute('d'));
 
-  var rect = this.svgPath_.getBoundingClientRect();
+  var rect = this.svgPath_.getBBox();
```

Comment:
For future reference, might also be worth peeking at `goog.style.getBounds` and `goog.style.getTransformedSize`